.cirrus.yml

```diff
@@ -78,4 +75,4 @@ test_ios_task:
   patrol_test_script: |
     cd packages/patrol/example
     flutter pub get
-    dart run patrol_develop_test.dart --device "$UDID"
+    dart run patrol_develop_test.dart
```

Comment:
I'm pretty sure that `dart patrol_develop_test.dart` is shorter and does the same